ui/selectors/selectors.js

```diff
@@ -601,6 +606,7 @@ export function getShowRecoveryPhraseReminder(state) {
 
 /**
  * To get the useTokenDetection flag which determines whether a static or dynamic token list is used
+ *
  * @param {*} state
```

Comment:
Is `*` the same as any? If so should this be standardized or does TypeDoc care?

Reply:
Lately we've been using TypeScript-style JSDoc types, so `any` would be preferred here. This is covered by the `valid-types` rule I think? Which is disabled in this PR because it would have required too much effort to fix all lint violations in this PR.

Reply:
Oh and to be clear, TypeDoc is only for TypeScript repositories. I want to use it for our TypeScript libraries, but it can't be used in this repository until after we migrate it to TypeScript. There are purely JSDoc doc generation projects that we could consider in the interim of course, but I haven't looked at those recently. I wanted to add lint rules here primarily to make the comments better for people reading the code directly, and to improve IDE support, and to make the later transition to TypeScript/TypeDoc easier.

Reply:
Ah ok, thanks for the clarification! I appreciate the incremental nature of these changes.